projects/1.16/assets/tinkers-construct/tconstruct/book/mighty_smelting/zh_cn/abilities/armor/tool_belt.json

```
@@ -0,0 +1,14 @@
+{
+  "modifier_id": "tconstruct:tool_belt",
+  "text": [
+    {
+      "text": "这是个用来放工具的腰带。工具，腰带。"
```

```suggestion
      "text": "这是个用来放工具的腰带。工具，腰带。是不是很好记？"
```
额……这里感觉像是在强调这是个“工具腰带”？也许可以这么改？